NETWORK/NetworkFadeInEntity.md

```diff
@@ -5,20 +5,33 @@ ns: NETWORK
 
 ```c
 // 0x1F4ED342ACEFE62D 0x9B9FCD02
-void NETWORK_FADE_IN_ENTITY(Entity entity, BOOL state);
+void NETWORK_FADE_IN_ENTITY(Entity entity, BOOL flash);
 ```
 
-```
-state - 0 does 5 fades  
-state - 1 does 6 fades  
-native is missing third argument, also boolean, setting to 1 made vehicle fade in slower, probably "slow" as per NETWORK_FADE_OUT_ENTITY  
-```
+Fade the given entity back in, usually used after the entity has been faded out with [NETWORK_FADE_OUT_ENTITY](#_0xDE564951F95E09ED)
+
+When used on a entity which isn't invisible or faded out then the native will still work, it will just instanly make the ped invisible before fading in.
+
+Note that there is another boolean parameter which is missing, when set to true the fade in will be a little bit slower.
 
-```
-NativeDB Added Parameter 3: BOOL slow
-```
 
 ## Parameters
-* **entity**: 
-* **state**: 
+* **entity**: The entity to fade in
+* **flash**: When set to true the entity will flash rapidly while fading out
+
+## Examples
+
+```lua
+NetworkFadeOutEntity(PlayerPedId(), false, false)
```

Comment:
`NetworkFadeOutEntity` has the boolean parameters reversed, first one is `bSendOverNet` and second one is `bFlash` or `flash`, I just append `b` behind it to specify it's a boolean.

I highlighted the diffs in red.
![diffs](https://github.com/citizenfx/natives/assets/38232208/3104ebf0-47d0-46a5-ae76-961435da5c7e)

What I also noticed is that fading an entity with occupants inside, such as a vehicle, will also fade the occupants, but only if they're classified as net objects (which is basically a net object method that sets the alpha for every entity within the seat manager instance of the specific vehicle entity)

![image](https://github.com/citizenfx/natives/assets/38232208/b3bbad6d-2209-4544-80b9-2c5cfcd29c34)

I improved your current example in order to demonstrate it:
```lua
RegisterCommand("fademe", function(source, args, rawCommand)
    CreateThread(function()
        local entity = GetVehiclePedIsIn(PlayerPedId())
        if entity == 0 then
            entity = PlayerPedId()
        else
            if IsVehicleSeatFree(entity, 0) then
                local playerDummy = ClonePed(PlayerPedId(), true, false, true)
                SetPedIntoVehicle(playerDummy, entity, 0)
            end
        end
       -- Last param is send over net, second is flash
        NetworkFadeOutEntity(entity, false, true)
        while NetworkIsEntityFading(entity) do
            Citizen.Wait(0)
        end
        -- Second parameter here is bSendOverNet, last one is flash
        NetworkFadeInEntity(entity, true, false)
        while NetworkIsEntityFading(entity) do
            Citizen.Wait(0)
        end
    end)
end, false)
```

Note: screenshots are provided for illustration/educational purposes only.